modules/site-health/audit-full-page-cache/helper.php

```diff
@@ -39,6 +39,8 @@ function perflab_afpc_get_page_cache_headers() {
 		'cf-cache-status'        => $cache_hit_callback,
 		'x-kinsta-cache'         => $cache_hit_callback,
 		'x-aruba-cache'          => $cache_hit_callback,
+		'x-litespeed-cache'      => $cache_hit_callback,
+		'x-qc-cache'             => $cache_hit_callback,
```

Comment:
You could add these as well:

```
x-proxy-cache
x-lsadc-cache
x-gateway-cache-status
```

Reply:
[x-gateway-cache-key](https://webtechsurvey.com/response-header/x-gateway-cache-key) ?